src/test/resources/features/cart.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Shopping Cart
+
+  Background:
+    Given Main page is opened
+
+  @Smoke
```

Comment:
You can define tags for the whole feature

Reply:
done